.pipelines/test-osm-arc-prod.yaml

```diff
@@ -0,0 +1,9 @@
+trigger: none
```

Comment:
Why is this file separate from .pipelines/osm-arc-prod.yaml ?

Reply:
The typical structure for ADO pipelines has been to have one yaml connected to the ADO job, which then calls another yaml which contains all the steps - ex: for the pr-e2e-arc, we have e2e-job, which then calls e2e-aks-arc.yaml. 

Maybe for this pipeline, I can centralize everything under test-osm-arc-prod.yaml, since there are fewer stages than the typical pr / release gate.

Reply:
Also I think a justification to split it into templates is so that parts can be reused, but I think in this case it's unlikely that a lot of this will be reused. If/when it needs to be reused though we could refactor it out into templates.